cmp/cmpopts/ignore.go

```diff
@@ -143,3 +144,58 @@ func isExported(id string) bool {
 	r, _ := utf8.DecodeRuneInString(id)
 	return unicode.IsUpper(r)
 }
+
+// IgnoreSliceElements returns an Option that ignores elements of []V.
+// The discard function must be of the form "func(T) bool" which is used to
+// ignore slice elements of type V, where V is assignable to T.
```

Comment:
Not clear from reading this whether returning true from the function means "ignore this" or "compare this".

Reply:
Done.